inc/admin.php

```diff
@@ -526,16 +526,26 @@ function enqueue_admin_scripts( $prefix = '', $force = false ) {
 		}
 	}
 
-	public function enqueue_yoast_compat(){
-        if( self::is_admin() && defined( 'WPSEO_FILE' ) && wp_script_is( 'yoast-seo-metabox' ) ) {
-            wp_enqueue_script(
-                'so-panels-yoast-compat',
-                siteorigin_panels_url( 'js/yoast-compat' . SITEORIGIN_PANELS_JS_SUFFIX . '.js' ),
-                array('jquery', 'yoast-seo-metabox' ),
-                SITEORIGIN_PANELS_VERSION,
-                true
-            );
-        }
+	public function enqueue_seo_compat(){
+		if ( self::is_admin() ) {
+			if ( defined( 'WPSEO_FILE' ) && wp_script_is( 'yoast-seo-metabox' ) ) {
+				wp_enqueue_script(
+					'so-panels-seo-compat',
+					siteorigin_panels_url( 'js/seo-compat' . SITEORIGIN_PANELS_JS_SUFFIX . '.js' ),
+					array('jquery', 'yoast-seo-metabox' ),
+					SITEORIGIN_PANELS_VERSION,
+					true
+				);
+			} else if ( defined( 'RANK_MATH_VERSION' ) && wp_script_is( 'rank-math-analyzer' ) ) {
```

Comment:
Probably best to switch to `elseif` to keep with the standard https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#use-elseif-not-else-if.